src/templates/Editorial/docs.md

```diff
@@ -1,4 +1,18 @@
-# Alle Elemente in der Standard Version
+```
+import createEditorialSchema from '@project-r/styleguide/lib/templates/Editorial'
+
+const schema = createEditorialSchema({
+  titleBlockAppend: <div>Share Actions</div>
+})
+```
+
+`createEditorialSchema` take an optional options object with follow keys:
+
+- `TitelBlockHeadline`, the title react component
```

Comment:
For conistency: Title... rather than Titel... 
Here and in
templates/Editorial/index.js
templates/Meta/index.js